src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -11,34 +11,51 @@ public class Exercise {
 
     public void stepOne() {
         // TODO: 1. Write a for loop that adds the numbers 0 to 3 to the numsZeroToThree array
-
+        numsZeroToThree = new int[4];
+        for(int i = 0; i < 4; i++)
+        {
+            numsZeroToThree[i] = i;
+        }
 
     }
 
     public void stepTwo() {
         // TODO: 2. Write a for loop that adds the numbers 5 to 10 to the numsFiveToTen array
-
+        numsFiveToTen = new int[6];
+        for(int i = 0; i < 6; i++)
+        {
+            numsFiveToTen[i] = i+5;
+        }
 
     }
 
     public void stepThree() {
         // TODO: 3. Write a for loop that adds the numbers 3 to 0 (in that order) to the countdown array
-
+        countdown = new int[4];
+        int num = 3;
+        for(int i = 0; i<3; i++)
+        {
+            countdown[i] = num;
+            num--;
+        }
```

Comment:
This can be simplified if you initialise your for loop on a different number. For example, `for(int i = 3;...`